test/parallel/test-repl-sigint-nested-eval.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+const spawn = require('child_process').spawn;
+
+process.env.REPL_TEST_PPID = process.pid;
+const child = spawn(process.execPath, [ '-i' ], {
+  stdio: [null, null, 2]
+});
+
+let stdout = '';
+child.stdout.setEncoding('utf8');
+child.stdout.on('data', function(c) {
+  process.stdout.write(c);
```

Comment:
Not wrong but you can accomplish the same thing with a `child.stdout.pipe(process.stdout)` call.